core/src/main/scala/com/databricks/labs/remorph/intermediate/ddl.scala

```diff
@@ -55,7 +57,7 @@ case object DayTimeIntervalType extends DataType
 
 // Complex types
 case class ArrayType(elementType: DataType) extends DataType
-case class StructField(name: String, dataType: DataType, nullable: Boolean = true)
+case class StructField(name: String, dataType: DataType, nullable: Boolean = true, metadata: Option[Metadata] = None)
```

Comment:
```suggestion
case class StructField(name: String, dataType: DataType, nullable: Boolean = true, metadata: Metadata = Metadata.empty)
```
This is will better implementation.

Reply:
I would rather introduce a Metadata Class Similar to spark definition. that can be reused across multiple use case like this

https://github.com/apache/spark/blob/master/sql/api/src/main/scala/org/apache/spark/sql/types/Metadata.scala#L29
 
 @nfx What are your thoughts? , if Yes this can go as a tech debt

Reply:
yep

Reply:
As disscussed i have made the necessary changes and made.

case class Metadata(comment: Option[String])